src/classes/RD2_Constants.cls

```diff
@@ -116,4 +116,8 @@ public inherited sharing class RD2_Constants {
     */
     public static final Integer DEFAULT_NEXT_DONATION_DATE_MATCH_RANGE_DAYS = 3;
 
+    public static final String RD_NAMING_DISABLE = 'Disable Auto Naming';
```

Comment:
If the default value of this field is null, is that sufficient to indicate that naming is disabled. The actual label in the picklist can come from `stgRDNamingDisable`, but the value to store would be null. If true, then you probably don't need a constant for this.

Also, do you think these constants really belong in the RD2_NamingService class?

Reply:
It makes sense to put the constant into RD2_NamingService. For the disable String, I was debating between empty string or auto populate the field with Disable the first time user save the RD setting in NPSP page if auto naming is off. I think empty strings makes more sense since it will be less conditions to check.